content/en/registry/instrumentation-js-sequelize.md

```diff
@@ -0,0 +1,15 @@
+---
+title: OpenTelemetry Sequelize Plugin
+registryType: instrumentation
+isThirdParty: false
```

Comment:
could you change this to 'true' as this isn't part of upstream otel?

Reply:
@austinlparker done.